vkconfig/mainwindow.cpp

```diff
@@ -1127,20 +1130,19 @@ bool MainWindow::eventFilter(QObject *target, QEvent *event) {
         }
     }
 
-    // If not configuration active, disable the setting area
+    // If no configuration active, disable the setting area
     // ui->groupBoxEditor->setEnabled(SaveLastItem() != nullptr);
```

Comment:
Dead code?

Reply:
Dang it. Yes, dead code. Copy and paste error from Christophe I think. Removed and recommited.